webextension/blank.html

```diff
@@ -0,0 +1 @@
+<html></html>
```

Comment:
QUIRKS MODE??!??!?!

Reply:
(kidding, this likely doens't matter)

Reply:
Heh. It's what screenshots is using, so I left it as-is :)